pm2.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apps": [
+    {
+      "name": "energy-2",
```

Comment:
Since the previous one has been deprecated, it is fine to name it `energy-frontend`.

Reply:
done